### PR TITLE
Update Linux and Windows platforms to Filament v1.9.19

### PR DESCRIPTION
**Pull request by errissa**
This change is [Reviewable](https://reviewable.io/reviews/intel-isl/open3d/3271)

### CODE REVIEW

**Review by yxlao — COMMENTED** [review]
LGTM to me after the Windows test is passing

File: 3rdparty/filament/filament_download.cmake [hunk 1]
```diff
@@ -1,25 +1,21 @@
...
     else()
         message(FATAL_ERROR "Filament binaries not found in ${FILAMENT_PRECOMPILED_ROOT}")
     endif()
 else()
-    set(lib_dir lib/x86_64)
+    set(lib_dir lib)
     # Setup download links
     if(WIN32)
-        set(DOWNLOAD_URL_PRIMARY "https://github.com/google/filament/releases/download/v1.9.9/filament-v1.9.9-windows.tgz")
+        set(DOWNLOAD_URL_PRIMARY "https://github.com/google/filament/releases/download/v1.9.19/filament-v1.9.19-windows.tgz")
```
[the remark this answers is not in the document]
Comment (yxlao) [reply 3rdparty/filament/filament_download.cmake:18]: i've invited you the `open3d_downloads` repo -- there should be an invitation email

[the remark this answers is not in the document]
Comment (errissa) [reply 3rdparty/filament/filament_download.cmake:18]: Should we put all platform binaries in the the new repository or just the ones we custom build and continue to download the non-custom builds from Filament github?

[the remark this answers is not in the document]
Comment (yxlao) [reply 3rdparty/filament/filament_download.cmake]: both works, perhaps just our own builds since there are less files to upload

**Review by yxlao — APPROVED** [review]

**Pull request merged**